examples/agent_services_example.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
```

Comment:
The name of the file does not need `example` - it is in the `examples` dir after all.

Reply:
fixed

Reply:
Changed, not fixed - still longer than 80 chars (some other lines in the file, too).

Reply:
Sorry, that comment was on the wrong change set...